quiz/result/result.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>lucent quiz</title>
+    <link rel="stylesheet" href="result.css">
+</head>
+
+<body>
+    <!-- Header Starts -->
+    <header class="header">
+        <div class="navbar-container">
+            <a class="navbar navbar-m-lr-1" href="../../home.html">
+                <h1 class="navbar-m-0">LUCENT QUIZ</h1>
+            </a>
+            <a class="navbar-m-left-auto navbar" href="#">
+                <h5>LOGIN</h5>
+            </a>
+            <a class="navbar navbar-m-lr-1 navbar-m-l-2" href="#">
+                <i class="fa-brands fa-github"></i>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-linkedin"></i>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+        </div>
+    </header>
+    <!-- Header Ends -->
+
+    <!-- Main Section Starts -->
+    <main class="main">
+        <div class="flex flex-direction">
+            <h2 class="text-center m-bottom">Result</h2>
+            <h3 class="text-center m-bottom">Final Score: 10</h3>
+            <div class="m-bottom">
+                <h4>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quod inventore sapiente repudiandae, quisquam culpa ut reprehenderit consectetur quos tenetur odio illo commodi magnam cumque voluptatibus, aperiam nesciunt, enim sint. Architecto.
+                </h4>
+                <button class="button">Answer 1</button>
+                <button class="button wrong-answer">Answer 2</button>
+                <button class="button correct-answer">Answer 3</button>
+            </div>
```

Comment:
fix: These would not be buttons as here you are showing the result to the user, here user would not be clicking in these